jobs/nfcens/stats/JEVS_NFCENS_WAVE_GRID2OBS_STATS

```diff
@@ -79,14 +73,9 @@ export jlogfile=${DATA}/${MODELNAME}_${RUN}_PointStat.log
 ####################################
 # Execute the script                
 ####################################
-env
-
-$HOMEevs/scripts/${COMPONENT}/${STEP}/exevs_${COMPONENT}_${RUN}_${VERIF_CASE}_${STEP}.sh 1>>$pgmout 2>errfile.${pid}
-
+$HOMEevs/scripts/nfcens/stats/exevs_nfcens_wave_grid2obs_stats.sh
```

Comment:
@lgannoaa , can you update to use $COMPONENT in place of nfcens, $STEP in place of stats? I would also like to see $RUN in place of wave and VERIF_CASE in place of grid2obs.